lib/config/config-validator.js

```diff
@@ -181,10 +186,10 @@ function formatErrors(errors) {
  * @returns {void}
  */
 function validateConfigSchema(config, source) {
-    const validator = schemaValidator(configSchema, { verbose: true });
+    validateSchema = validateSchema || ajv.compile(configSchema);
```

Comment:
I'm not sure we ever need that. It's just my personal preferences. Prob no need to do that at all ;-)